sdk/communication/communication-identity/swagger/README.md

```diff
@@ -11,8 +11,8 @@ description: Communication identity client
 package-version: 1.3.2
```

Comment:
Before we release we will also have to 
* update the version here, in constants.ts and package.json
* update the CHANGELOG
* update the README
* update tests